openrr-planner/src/collision/self_collision_checker.rs

```diff
@@ -175,12 +187,16 @@ fn test_create_self_collision_checker() {
     assert!(self_collision_checker
         .check_joint_positions(
             &[0.0; 16],
-            &[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.57, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+            &[1.57, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
             std::time::Duration::new(1, 0),
         )
         .is_err());
 
-    let l_shoulder_yaw_node = robot.find("l_shoulder_yaw").unwrap();
+    let l_shoulder_yaw_node = self_collision_checker
+        .robot_collision_detector
+        .robot
+        .find("l_shoulder_yaw")
+        .unwrap();
```

Comment:
I want to know why you changed the test. Something has been changed by this change? or just improve something?

Reply:
The purposes of the change for this test are:

- to make "sample.urdf" a variable (for improvement)
- to correct collision checking (since the order of the joints is somehow changed...)

Reply:
Thanks! I understood that this change causes some shuffling  (it might be making it go the correct direction) about the order of joints comparing the older code.